src/client/article.ts

```diff
@@ -347,8 +346,7 @@ function richLinks(): void {
 						if (placeholder && image) {
 							placeholder.innerHTML = `<img src="${image}" alt="related article"/>`;
 						}
-					},
-				);
+					});
```

Comment:
We may want a `catch` to avoid crashing the client JS? For example, if the user is offline this request will fail and there won't be any JSON in the response. Perhaps we should check the response code?